skimage/exposure/_adapthist.py

```diff
@@ -36,7 +36,7 @@ def equalize_adapthist(image, kernel_size=None,
     ----------
     image : (M, N[, C]) ndarray
         Input image.
-    kernel_size: integer or list-like, optional
+    kernel_size: array_like, optional
```

Comment:
```suggestion
    kernel_size: integer or array_like, optional
```

Reply:
I don't think integer fall into the realm of `array_like`

Reply:
According to [numpy doc guidelines](https://numpydoc.readthedocs.io/en/latest/format.html#other-points-to-keep-in-mind), it does :-)

Reply:
ok, so you are right but I still find this confusing :-). I would expect most people to be as wrong as me ;-) and think they must provide some kind of iterable. I might do a twitter poll just for the fun of it ;-). Don't worry this will not block the PR.

Reply:
https://twitter.com/EGouillart/status/1225465212398194691 ;-)